re/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "esy-bash",
+  "version": "0.1.0",
+  "description": "Reason workflow with Esy",
+  "license": "MIT",
+  "esy": {
+    "build": "refmterr dune build -p #{self.name}",
+    "NOTE": "Optional release Section. Customizes result of `esy release`",
+    "release": {
+      "releasedBinaries": [
+        "EsyBash.exe"
+      ],
+      "deleteFromBinaryRelease": [
+        "*esy_ocaml_*",
+        "*refmterr-*",
+        "ocaml-*"
+      ]
+    }
+  },
+  "dependencies": {
+    "@opam/dune": "*",
+    "@opam/lambda-term": "*",
+    "@opam/lwt": "*",
+    "@esy-ocaml/reason": "*",
+    "refmterr": "*",
+    "ocaml": "~4.6.0"
+  },
```

Comment:
I think we can remove unused deps from here.

Reply:
I'm not entirely sure which of the above are unnecessary. Can you please point them out?

Reply:
Looks like `@opam/lwt` and `@opam/lambda-term` could be removed here.

Reply:
Oh yes, my bad. Clearly not needed.